Sources/StreamChatUI/Composer/ComposerKeyboardHandler.swift

```diff
@@ -63,10 +63,20 @@ open class ComposerKeyboardHandler: KeyboardHandler {
               let frame = (userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue,
               let duration = userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? Double,
               let curve = userInfo[UIResponder.keyboardAnimationCurveUserInfoKey] as? UInt,
-              let composerParentView = composerParentVC?.view else {
+              let composerParentView = composerParentVC?.view,
```

Comment:
Small detail that doesn't really change anything - only one unwrapping is needed here now, for the composerParentVC.
```
guard composerParentVC?.presentedViewController == nil,
              let userInfo = notification.userInfo,
              let frame = (userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue,
              let duration = userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? Double,
              let curve = userInfo[UIResponder.keyboardAnimationCurveUserInfoKey] as? UInt,
              let composerParentVC = composerParentVC else {
            return
        }
        
        let composerParentView = composerParentVC.view
```

Reply:
It's actually true because the view from the view controller is also optional, so we need both 👍

Reply:
You are right, disregard my comment 😁